src/objecttypes/conf/base.py

```diff
@@ -10,10 +10,10 @@
 #
 
 # Application definition
-
 INSTALLED_APPS = INSTALLED_APPS + [
     # External applications.
     "jsonsuit.apps.JSONSuitConfig",
+    "open_api_framework",
```

Comment:
I think it would be better to add this to `INSTALLED_APPS` in `open_api_framework/conf/base.py` itself :)